basic-application/basic-application-core/src/main/java/fr/openwide/core/basicapp/core/business/referential/dao/CityDaoImpl.java

```diff
@@ -0,0 +1,24 @@
+package fr.openwide.core.basicapp.core.business.referential.dao;
+
+import org.springframework.stereotype.Repository;
+
+import com.querydsl.jpa.impl.JPAQuery;
+
+import fr.openwide.core.basicapp.core.business.referential.model.City;
+import fr.openwide.core.basicapp.core.business.referential.model.QCity;
+import fr.openwide.core.jpa.business.generic.dao.GenericEntityDaoImpl;
+
+@Repository("cityDao")
+public class CityDaoImpl extends GenericEntityDaoImpl<Long, City> implements ICityDao {
+
+	private final QCity qCity = QCity.city;
+	
+	@Override
+	public City getByUniqueKeyIgnoreCase(String libelle) {
```

Comment:
You should rather use `fr.openwide.core.jpa.more.business.generic.dao.IGenericListItemDao.getByFieldIgnoreCase(EntityPath<T>, StringExpression, String)`.